lib/internal/bootstrap_node.js

```diff
@@ -39,13 +39,13 @@
     _process.setup_cpuUsage();
     _process.setupMemoryUsage();
     _process.setupConfig(NativeModule._source);
-    NativeModule.require('internal/process/warning').setup();
-    NativeModule.require('internal/process/next_tick').setup();
-    NativeModule.require('internal/process/stdio').setup();
+    NativeModule.require('internal/process/warning')();
+    NativeModule.require('internal/process/next_tick')();
+    NativeModule.require('internal/process/stdio')();
     _process.setupKillAndExit();
     _process.setupSignalHandlers();
     if (global.__coverage__)
-      NativeModule.require('internal/process/write-coverage').setup();
+      NativeModule.require('internal/process/write-coverage')();
```

Comment:
these are already different from the other ones tho and we eliminate an extraneous property lookup. I'm not going to fight for it, but I don't see much value in keeping the separate `setup()` functions.